src/cms/templates/_base.html

```diff
@@ -145,7 +145,7 @@
 					<a href="{% url 'pages' region_slug=region.slug %}" class="{% if current_menu_item == 'pages' %}active{% endif %}">
 						{% trans 'All pages' %}
 					</a>
-					{% if region.default_language == language and language != null %}
+					{% if language %}
```

Comment:
```suggestion
					{% if region.default_language == language %}
```
The condition in its original form was added on purpose, because it should prevent to create new pages in other languages than the default language.
So the "Create Page"-button should only appear if the current language is the default language. But the second condition was redundant I think.